### PR TITLE
export validator.isValidating

### LINKED ISSUE #386: export isValidating in web3.ext
### _Type_:
Request

### _What happened? / What do you need?_:
`isRunning` can't say much about validator's state.

### _Priority_:
Medium

### PR DESCRIPTION
**Pull request by JekaMas**
Important changes:
- [x] adds `validator.isValidating()`

Closes #386

### CODE REVIEW

File: internal/web3ext/web3ext.go [hunk 1]
```diff
@@ -495,6 +495,10 @@ web3._extend({
 			name: 'isRunning',
 			call: 'validator_isRunning'
 		}),
+		new web3._extend.Method({
+			name: 'isValidating',
```
Comment (acroca) [inline internal/web3ext/web3ext.go]: What's the usecase of `validator.isRunning` vs `validator.isValidating`? As a user of `validator`, when should I use either one? They sound similar, like if a validator object is running for me it means it's validating, that's why it's called validator... no?

Comment (JekaMas) [reply internal/web3ext/web3ext.go]: Afaik validator can be in 2 states: running and validating.
If validator is 'running' it doesn't mean that it is 'validating'. But if it's validating, it's running also.
Maybe we should remove isRunning?

Comment (acroca) [reply internal/web3ext/web3ext.go]: I don't know. I don't understand why a validator would be running and not validating. But if that's a thing, then it's alright.

Comment (jmatosp) [reply internal/web3ext/web3ext.go]: I think running means that is trying to start validating. the user says start, but hasn't yet joined the validators

Comment (acroca) [reply internal/web3ext/web3ext.go]: I see. Thanks @jmatosp

Comment (jmatosp) [reply internal/web3ext/web3ext.go]: but, not easy to follow :|

**Pull request merged**